### DISCUSSION TITLE
Nest dialog inside __next.

### DISCUSSION THREAD

**Discussion by VaibhavAcharya** [radix-ui/primitives#514] [Help]
By default `dialog` is nested inside `document.body` but I would like it to be inside `#__next` tag.
Is there any way to do it ?

**Comment by benoitgrelard** [comment]
There's currently no easy way to do this (although we have some internal mechanics but are not exposed).
For us to consider your use case, could you please describe what issue you are facing and why you need this?

Thanks!
